psalm.xml

```diff
@@ -24,6 +27,20 @@
             </errorLevel>
         </StringIncrement>
 
+        <ImplicitToStringCast>
+            <errorLevel type="suppress">
+                <directory name="examples"/>
+                <directory name="src"/>
+            </errorLevel>
+        </ImplicitToStringCast>
+
+        <InvalidOperand>
```

Comment:
Don't suppress this at a project level

Reply:
https://github.com/vimeo/psalm/pull/11329 will fix what you're (incorrectly) trying to fix

Reply:
It's not only that. The code often do some `float * int`, which is perfectly fine, btw `string . float` is also ok.

Reply:
Psalm is capable of running scans for versions down to php 5.4 on php 8.4, just pass the correct PHP version using the `--php-version` parameter.

Reply:
Psalm scans should **always** run on PHP 8.4.4 or above to avoid platform bugs, to target lower versions of PHP simply use the `--php-version` parameter.